src/components/workspace.tsx

```diff
@@ -89,7 +94,10 @@ export class WorkspaceComponent extends BaseComponent<IProps, {}> {
   }
 
   private renderToolbar() {
-    const { workspace } = this.props;
+    const workspace = this.props.workspace;
+    if (!workspace.tool) {
```

Comment:
I chose to set the `tool` in `PublishedWorkspaceModel` to `undefined`, rather than switch on the workspace type here. Where possible, I prefer having the workspace behaviors described in the workspace definitions rather than scattered throughout the code, to leave fewer surprises if an additional workspace is added in the future.